reference-architecture/gce-ansible/playbooks/openshift-setup.yaml

```diff
@@ -9,19 +9,20 @@
       cloud-provider:
       - "gce"
       cloud-config:
-      - "/etc/origin/gce.ini"
+      - "/etc/origin/cloudprovider/gce.conf"
```

Comment:
Instead of configuring osm_controller_args for the cloud provider, should be using the openshift-ansible provided config.